spec/features/02_user_can_see_a_cryptid_show_page_spec.rb

```diff
@@ -8,11 +8,15 @@
   #Acceptance Criteria
   # * If I go to the cryptid show page I see info about the cryptid
   let!(:test_user) { FactoryBot.create(:user) }
+  let!(:usa) { Region.create!(name: "USA") }
+  let!(:reptile) { Category.create!(name: "Reptile") }
   let!(:cryptid) do
      Cryptid.create!(
       name: "Dragon",
       description: "Breathes fire",
-      user: test_user)
+      user: test_user,
+      region: usa,
+      category: reptile)
   end
 
   scenario 'user visits show page' do
```

Comment:
I should see more information about cryptids on this page including the description and the region and the categories